src/UI/WB.UI.Frontend/src/hqapp/Views/HQ/Export/ExportProcessCard.vue

```diff
@@ -209,6 +218,25 @@ export default {
                 }
             })
         },
+        getProducedTime(data){
+            if(data == undefined || data.endDate == undefined || data.beginDate == undefined)
+                return 'unknown';
+
+            let diff = moment(data.endDate).diff(moment(data.beginDate));
+            let duration = moment.duration(diff);
+            
+            return duration.humanize({m: 60, h: 24, d: 7, w: 4});
+        },
+        getInQueueTime(data){

```

Comment:
getProducedTime and getInQueueTime are similar please extract to separate method common part